Userland/Services/FileOperation/main.cpp

```diff
@@ -87,7 +104,7 @@ static bool collect_work_items(const String& source, const String& destination,
     Core::DirIterator dt(source, Core::DirIterator::SkipParentAndBaseDir);
     while (dt.has_next()) {
         auto name = dt.next_path();
-        if (!collect_work_items(
+        if (!collect_copy_work_items(
                 String::formatted("{}/{}", source, name),
                 String::formatted("{}/{}", destination, LexicalPath(source).basename()),
```

Comment:
Not your code, but this should use `LexicalPath::join`, because otherwise, if `source` or `destination` happened to be `/`, you would get `//foo.txt` (and there are also some other shenanigans which can happen when hand-crafting paths.

Reply:
Done!